go.mod

```diff
@@ -37,6 +37,88 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require (
```

Comment:
It's interesting this was added in the go 1.18 upgrade. AFAICT, this separate block for indirect dependencies was [introduced](https://go.dev/doc/go1.17#go-command) in go 1.17...

Reply:
Yep , on the go.mod we jumped from 1.16 to 1.18  :-)

Reply:
🤦

Reply:
the Other docker images were on 1.17  !! , now everything is consistent , 
is it the best practice to be on the latest go compiler for builds and the go version lags one version on the module?

Reply:
no - this was an oversight. thanks for correcting!